packages/api/src/external/carequality/command/cq-directory/__tests__/search-cq-directory.test.ts

```diff
@@ -0,0 +1,73 @@
+import { faker } from "@faker-js/faker";
+import convert from "convert-units";
+import { makeAddressWithCoordinates } from "../../../../../domain/medical/__tests__/location-address";
+import { makePatient } from "../../../../../domain/medical/__tests__/patient";
+import { CQDirectoryEntry } from "../../../cq-directory";
+import { makeCQDirectoryEntry } from "../../../__tests__/cq-directory";
+import {
+  searchCQDirectoriesAroundPatientAddresses,
+  searchCQDirectoriesByRadius,
+} from "../search-cq-directory";
+
+describe("searchCQDirectoriesAroundPatientAddresses", () => {
+  let searchCQDirectoriesByRadius_mock: jest.Func;
+  beforeAll(() => {
+    jest.restoreAllMocks();
+  });
+  beforeEach(() => {
+    searchCQDirectoriesByRadius_mock = jest.fn(
+      async () => []
+    ) as typeof searchCQDirectoriesByRadius;
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  afterAll(() => {
+    jest.restoreAllMocks();
+  });
+
+  it("returns empty array if no coordinates", async () => {
+    const patient = makePatient();
+    const result = await searchCQDirectoriesAroundPatientAddresses({ patient });
```

Comment:
if you arent passing in the `searchCQDirectoriesByRadius_mock` here, then we dont know if the test is passing because the patient has no coordinates or because the `searchCQDirectoriesByRadius_mock` isnt even available to `searchCQDirectoriesAroundPatientAddresses`

Reply:
True! Will update.